develop-docs/sdk/telemetry/logs.mdx

```diff
@@ -299,6 +299,8 @@ If the log was paramaterized the SDK should attach the following
 }
 ```
 
+If there are no `sentry.message.parameter.X` attributes included in the log, then the SDK MUST NOT attach a `sentry.message.template` attribute.
```

Comment:
Can you add that this is important for PII reasons, since lacking `parameter` means that template isn't a template but a clone of body?